### PR TITLE
Move BvInverter class into separate file.

### PR DESCRIPTION
**Pull request by mpreiner**

### CODE REVIEW

File: src/theory/quantifiers/bv_inverter.cpp [hunk 1]
```diff
@@ -0,0 +1,314 @@
+/*********************                                                        */
+/*! \file ceg_t_instantiator.h
```
Comment (ajreynol) [inline src/theory/quantifiers/bv_inverter.cpp]: This should be bv_inverter.cpp

File: src/theory/quantifiers/bv_inverter.cpp [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief theory-specific counterexample-guided quantifier instantiation
+ **/
```
Comment (aniemetz) [inline src/theory/quantifiers/bv_inverter.cpp]: Update header.

File: src/theory/quantifiers/bv_inverter.h [hunk 3]
```diff
@@ -0,0 +1,135 @@
+/*********************                                                        */
+/*! \file ceg_t_instantiator.h
```
Comment (ajreynol) [inline src/theory/quantifiers/bv_inverter.h]: This should be bv_inverter.h

File: src/theory/quantifiers/bv_inverter.h [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief theory-specific counterexample-guided quantifier instantiation
+ **/
```
Comment (aniemetz) [inline src/theory/quantifiers/bv_inverter.h]: Ditto.

**Pull request merged**